cmd/upgrade/main.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2018 The OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"os"
+
+	log "github.com/golang/glog"
+	upgrade "github.com/openebs/maya/cmd/upgrade/app/v1alpha1"
+)
+
+func main() {
+	flag.Set("logtostderr", "true")
+	configPath := flag.String("config-path", "/config/config", "Path to Job config file.")
```

Comment:
What is the standard config path for any binary. Can it be `/etc/config/upgrade` ?

Reply:
Please check if message provided to flag.String(...) can be all small letters. Follow the standard.

Reply:
Let us avoid the term Job. Since Job is external to this app/binary.
It can run as a binary or something that is not a job.